core/storages.py

```diff
@@ -0,0 +1,22 @@
+import boto3
+
+class S3Client:
+    def __init__(self, access_key, secret_key):
+        self.s3 = boto3.client(
+            's3',
+            aws_access_key_id     = access_key,
+            aws_secret_access_key = secret_key
+        )
+
+    def upload(self, file, file_name, bucket_name):
+        self.s3.upload_fileobj(
+                file,
+                bucket_name,
+                file_name,
+                ExtraArgs={
+                    "ContentType": file.content_type
+                }
+            )           

```

Comment:
하나의 업로드 기능인데 필요한 로직이 밖에 존재합니다. method 안에 넣어주세요.
```suggestion
    def upload(self, file, bucket_name):
        # file_name 만드는 로직 추가
        self.s3.upload_fileobj(
                file,
                bucket_name,
                file_name,
                ExtraArgs={
                    "ContentType": file.content_type
                }
            )           

```

Reply:
수정했습니다.